content/birds/sondra.md

```diff
@@ -32,3 +34,36 @@ that reached over 200% of their goal.
 Sondra is also a multimedia artist, author, and musician –
 bringing a unique perspective
 to audience engagement and interaction.
+
+<div class="contain">
+
+{{ embed.img(
+  src='birds/lorescertificationbadge.png',
+  alt='certified object-oriented ux',
+    attrs={
+    'class': 'align-left'
+  }
+) }}
+
+In 2020, Sondra became a Certified Object-Oriented UX Strategist.
+
+</div>
+
+OOUX Strategists can:
```

Comment:
Love the list, don't love the title - especially "can". Maybe we can chat about alternatives tomorrow.